Challenge-06 anshitpoddar

anshitpoddar opened the pull request.

TeralaShivaKumar commented:
Hello anshitpoddar please squash all commits

TeralaShivaKumar commented:
hello [anshitpoddar](https://github.com/anshitpoddar)
Squash commits to eligible for rewards

anshitpoddar commented:
> hello [anshitpoddar](https://github.com/anshitpoddar) Squash commits to eligible for rewards

I have squashed my following commits....please check it once and let me know..

TeralaShivaKumar commented:
Hello [anshitpoddar](https://github.com/anshitpoddar)
It is still showing 29 commits please squash properly

anshitpoddar commented:
> Hello [anshitpoddar](https://github.com/anshitpoddar) It is still showing 29 commits please squash properly

Hello there, I already squashed all my commits yesterday and it successfully happened too, when I entered the rebase CLI, it showed me some commits and I picked the very first commit and squashed the rest and it worked fine, I don't understand why is it showing 29 commits. Could you please specify what exactly I have to do? I am having trouble figuring it out.

anshitpoddar commented:
> Hello [anshitpoddar](https://github.com/anshitpoddar) It is still showing 29 commits please squash properly

Hey there... Actually in my forked repo I have discarded my 29 commits and hence these commits were discarded from my PR and PR is closed now....what can I do??
Please do help... @TeralaShivaKumar
